src/index.html

```diff
@@ -7,8 +7,51 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <!-- fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin=" "
+      >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo" href="#home">
+        <img src="images\logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="link is-active" href="#apple">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="link" href="#samsung">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="link" href="#smartphones">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+            class="link"
+            href="#laptops"
+            data-qa="hover"
+            >Laptops & Computers</a>
+          </li>
```

Comment:
```suggestion
          <li class="nav__item">
            <a
              class="link"
              href="#laptops"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
          </li>
```

Reply:
**Added links to the PR desc**

- [DEMO LINK](https://reyti.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://reyti.github.io/layout_moyo-header/report/html_report/)